Rubberduck.Core/Rubberduck.Core.csproj

```diff
@@ -123,9 +123,10 @@
       <AutoGen>True</AutoGen>
       <DesignTimeSharedInput>True</DesignTimeSharedInput>
       <DependentUpon>Settings.settings</DependentUpon>
+      <DesignTime>True</DesignTime>
     </Compile>
     <None Include="Properties\Settings.settings">
-      <Generator>PublicSettingsSingleFileGenerator</Generator>
+      <Generator>SettingsSingleFileGenerator</Generator>
```

Comment:
This change wasn't made by me directly. I am not sure if that's a good thing or not.

Reply:
might be an issue. The SettingsSingleFileGenerator just generates internal-visible code. Especially for Rubberduck.Resources that change would break compilation. If it doesn't break compilation, that should be fine

Reply:
I see. I wonder how that got changed as I don't remember fiddling with that. It did build and run tests locally, however. I'll need to actually F5 test it tonight, though.

Reply:
If it builds, then it's not an issue to have that change. I assume that it correctly defaults to least visibility modifying the resource itself. Since you changed the file that's involved here, VS automatically pushed it into the lower visibility on build